docker-hub/download-rate-limit.md

```diff
@@ -40,14 +40,14 @@ You will see this error message in the Docker CLI or in the Docker Engine logs.
 
 ## How can I check my current rate
 
-When limiting starts, valid non-rate-limited manifest API requests to Hub will include the following rate limit headers in the response:
+Valid non-rate-limited manifest API requests to Hub will include the following rate limit headers in the response:
```

Comment:
Is this clearly worded? "Non-rate-limited" almost sounds like it could be talking about if you are an unlimited pulls user.